src/radical/pilot/pmgr/launching/default.py

```diff
@@ -125,6 +132,10 @@ def _pmgr_control_cb(self, topic, msg):
         if cmd == 'pilot_staging_input_request':
 
             self._handle_pilot_input_staging(arg['pilot'], arg['sds'])
+            self.publish(rpc.CONTROL_PUBSUB,
+                         {'cmd': 'pilot_staging_input_result',
```

Comment:
this command `pilot_staging_input_result` is not read anywhere else

Reply:
also fixed.